src/HybridModelBinding/HybridMvcBuilderExtensions.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using HybridModelBinding;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace Microsoft.Extensions.DependencyInjection
```

Comment:
I'm okay with this for now since the `Startup` will probably already be `using` this namespace.

Reply:
Yep. `Startup` will **necessarily** be using this namespace because `.AddMvc()` method already use this namespace.